packages/common-all/src/dnode.ts

```diff
@@ -870,7 +882,10 @@ export class NoteUtils {
       vault: toVault,
     });
 
-    if (maybeNotes.length > 1) {
+    if (maybeNotes.length === 1) {
```

Comment:
can we refactor the signature here? this function calls ` NoteUtils.getNotesByFname`. we should use `getNotesByFnameFromEngine` instead

Reply:
sure